internal/consensus/state_test.go

```diff
@@ -622,6 +622,8 @@ func TestStateLockPOLRelock(t *testing.T) {
 	require.NoError(t, err)
 	addr := pv1.Address()
 	voteCh := subscribeToVoter(cs1, addr)
+	unlockCh := subscribe(cs1.eventBus, types.EventQueryUnlock)
```

Comment:
Would be good if we can git rid of Unlock event all together, but probably you want to do it in different PR.